angular-clipboard.js

```diff
@@ -48,7 +80,12 @@ return angular.module('angular-clipboard', [])
                     $event.preventDefault();
                   }
 
-                  if (options && options.copyAsUrl) {
+                  // On chrome+edge, if we set text/uri-list and text/plain, the text/plain does not come across
+                  // upon paste. This is needed since it seems that more programs / apps rely on text/plain.
+                  // This seems like a chrome bug.
+                  // No issues on safari and firefox.
+                  // So for now, do not set the text/uri-list for now for chrome and edge
+                  if (options && options.copyAsUrl && (_browser !== 'chrome') && (_browser !== 'edge')) {
```

Comment:
Why do we want to se the text/uri-list at all? What is the benefit? Would it be better just to set plain/text for all? It is not as if we provide friendly names with the uri-list.

Reply:
https://jira.bublup.com/browse/NGM-3437 - text/uri-list was added because of this request.